store/sql_post_store.go

```diff
@@ -212,6 +212,58 @@ func (s SqlPostStore) GetFlaggedPosts(userId string, offset int, limit int) Stor
 	return storeChannel
 }
 
+func (s SqlPostStore) GetFlaggedPostsForTeam(userId, teamId string, offset int, limit int) StoreChannel {
+	storeChannel := make(StoreChannel, 1)
+	go func() {
+		result := StoreResult{}
+		pl := model.NewPostList()
+
+		var posts []*model.Post
+
+		query := `
```

Comment:
I commented on this days ago and it disappeared..

Well the gist of what I said was:

Not too worried about performance since it's a rare query also I think doing an inner join on a small subset of of the Posts table is better than doing two full queries on the Posts table. Plus I'm sure the database engine will do a bunch of query optimization anyways